scalableminds/webknossos
# Enable merger mode in skeleton and hybrid tracings

### URL of deployed dev instance (used for testing):
- [https://enablemergermodeinhybridtracings.webknossos.xyz](https://enablemergermodeinhybridtracings.webknossos.xyz)

### Steps to test:
- open a skeleton or hybrid tracing with a segmentation layer
- create some trees
- then open the settings menu and enable the merger mode under "Nodes & Trees"
- the segments that are marked with a node of a tress should now be colored in the same color
- further annotating should all color all annotated segments in the same color (for each tree)
- pressing 8 should ask the user whether to change the color of the segment. If selected yes a random color for the segments should be chosen. Selecting no should do nothing.
- pressing 9 should turn the opacity of the segmentation on and off.
- disabling the merger mode again should turn off all these features but the segments should keep their color

### Issues:
- fixes #3351

------
